blockstack/blockstackd.py

```diff
@@ -24,6 +24,7 @@
 import os
 import sys
 import signal
+import pipes
```

Comment:
Is this part of the standard Python base installation?  If not, can you add a line for it in `setup.py`?

Reply:
Yes, it's a part of the python std lib https://docs.python.org/2/library/pipes.html (unless I'm mistaken about what the python base installation includes)